src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/ApiExceptionHandler.kt

```diff
@@ -313,6 +326,9 @@ class ApiExceptionHandler {
 }
 
 class LocationNotFoundException(id: String) : Exception("There is no location found for ID = $id")
+
+class LocationPrefixNotFoundException(id: String) : Exception("No mappings found for $id")
```

Comment:
Think message could be clearer

Reply:
will change to "Location prefix not found for $id"